### PR TITLE
feat(curriculum): remove space requirement in steps 43&46 of cat photo app project

### LINKED ISSUE #46146: Improve Step 43 & 46 because of confusion?
**Describe the bug**
I dont know if this is supposed to be like this but in the new Responsive Web Design Course in step 43 and also in step 46 when the radio buttons are introduced, you need to have a space between the input element and the text 'Indoor' and 'Outdoor'.

Wrong: 
grafik

Right: 
grafik

Maybe this should be changed, since it did confuse me quiet a bit.

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46146

Maybe something like this? so it mentions the step

### CODE REVIEW

**Review by Sboonny — CHANGES_REQUESTED** [review]
I am in support of @RandellDawson [comment](https://github.com/freeCodeCamp/freeCodeCamp/issues/46146#issuecomment-1137467569)

The spaces are mainly used for styling as a way to add margin. We should not force people to try to style a page with HTML, if they want to do it themselves, then they have themselves to blame

**Comment by ieahleen** [comment]
I would argue that it's common practice to treat inline elements, like `input`, as part of the text, so the space is not just styling.

**Comment by Sboonny** [comment]
@ieahleen sorry it hasn't reached my brain. 

Is it common for `input` to be treated like an anchor `a` as, they can be wrapped around text like in 

```html
<p>Click here to view more <a target="_blank" href="https://freecatphotoapp.com">cat photos</a>.</p>
```

but for labels?

**Comment by ieahleen** [comment]
Yes, input elements are inline elements, so they are like a word

**Review by Sboonny — DISMISSED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Comment by ieahleen** [comment]
ok, fine, the posts on the forum have convinced me - let's remove the space requirement.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804dc.md [hunk 1]
```diff
@@ -125,6 +125,7 @@ assert(
         <h2>Cat Form</h2>
         <form action="https://freecatphotoapp.com/submit-cat-photo">
 --fcc-editable-region--
+
```
Comment (RandellDawson) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804dc.md:128]: Why is this extra line being added here?

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5f05a1d8e233dff4a68508d8.md [hunk 2]
```diff
@@ -97,6 +97,7 @@ assert($('input')[1].id.match(/^outdoor$/));
         <form action="https://freecatphotoapp.com/submit-cat-photo">
 --fcc-editable-region--
           <label><input id="indoor" type="radio"> Indoor</label>
+
```
Comment (RandellDawson) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5f05a1d8e233dff4a68508d8.md:100]: Why is this extra line being added here?

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804dc.md (hunk 1 above)
Comment (ieahleen) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804dc.md:128]: it looks better

Comment (RandellDawson) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804dc.md:128]: I guess that is a matter of opinion.  The next step does not show an extra blank line here.  This is not typically done on other steps in the project, so why now?

Comment (ieahleen) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804dc.md:128]: it's the space for the user to write in, considering it's the first project, showing a blank line where the user can write may cause less confusion for the user

but effectively it's not there anywhere else

Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804dc.md:128]:
```suggestion
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5f05a1d8e233dff4a68508d8.md (hunk 2 above)
Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5f05a1d8e233dff4a68508d8.md:100]:
```suggestion
```

**Review by Sboonny — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**